.golangci.yml

```diff
@@ -38,10 +38,21 @@ linters:
 issues:
   max-same-issues: 0
   max-issues-per-linter: 0
+  # We are disabling default golangci exclusions because we want to help reviewers to focus on reviewing the most relevant
+  # changes in PRs and avoid nitpicking.
+  exclude-use-default: false
   # List of regexps of issue texts to exclude, empty list by default.
   exclude:
     - Using the variable on range scope `(tc)|(rt)|(tt)|(test)|(testcase)|(testCase)` in function literal
     - "G108: Profiling endpoint is automatically exposed on /debug/pprof"
+    # The following are being worked on to remove their exclusion. This list should be reduced or go away all together over time.
+    # If it is decided they will not be addressed they should be moved above this comment.
+    - (comment on exported (method|function|type|const)|should have( a package)? comment|comment should be of the form)
+    - package comment should be of the form "Package (.+) ..."
+    - Error return value of .((os\.)?std(out|err)\..*|.*Close|.*Flush|os\.Remove(All)?|.*print(f|ln)?|os\.(Un)?Setenv). is not checked
```

Comment:
I'd like to keep these. For the `os` package, in almost all cases, calls are made using syscalls; having these in a `defer` call is usually ideal, and while ignoring the error might seem odd, if one is returned it's usually a problem that would have been caught earlier on (like when opening a file, or trying to write to it)

Reply:
Would you prefer I move this up now? Or wait to investigate first?

> If it is decided they will not be addressed they should be moved above this comment.

Reply:
I'd move it, yeah

Reply:
Done